getting_started/workflow/assets/importing_images.rst

```diff
@@ -149,6 +149,9 @@ HDR as SRGB
 
 Some few HDR files are broken and contain SRGB color data. It is advised to not use them but, in the worst case, toggling this option on will make them look right.
 
+Invert Color
+~~~~~~~~~~~~
+When this option is enabled, the colors of the image will be reversed. This option is useful for example to convert a height map generated by external programs to depth map to use with :ref:`doc_spatial_material`.
```

Comment:
Add a blank line between titles and paragraphs, and please avoid using the passive voice: http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html